<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDatabaseClass</class>
 <widget class="QWidget" name="ChangeDatabaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeDatabase</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="exisitngDBRadio">
      <property name="text">
       <string>Existing database</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="createNewRadio">
      <property name="text">
       <string>Create new</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnConnectDB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	border-radius: 4px;
	background-color: rgba(70, 133, 210, 1);
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/ADMIN/Documents/icons/database.png</normaloff>C:/Users/ADMIN/Documents/icons/database.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="sqliteRadio">
      <property name="text">
       <string>SQLite</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="mysqlRadio">
      <property name="text">
       <string>MySQL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Database file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pathDBLineEdit">
      <property name="maximumSize">
       <size>
        <width>189</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBrowseFile">
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	border-radius: 4px;
	background-color: rgba(70, 133, 210, 1);
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="hostNameLabel">
      <property name="text">
       <string>Host name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hostNameLineEdit">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="dbNameLabel">
      <property name="text">
       <string>Database Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dbNameLineEdit">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameLineEdit">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>310</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
